### ISSUE TITLE
React Docs

### ISSUE THREAD

**Issue by nadnoslen** [miragejs/site#58]
Hey Gang,

Thanks for this port.  I'm an Ember dude stuck doing some React stuff and brought this over into my project.  Awesome ... love it!

For us, I folded mirage into our React test environment.  While it's fresh in my mind, did you want me to create fork of your site with some documentation for that React examples page:  https://miragejs.com/examples/main/react

If you do want the help, let me know if you had any high-level expectations about what topics it would cover.  I was thinking:

* Basic Configuration including example Model, Factory, Route
* Some Faker.js
* Some jest test examples

I do realize that some of this example info already lives here: https://github.com/miragejs/react-demo ... but it would be nice to break it down on the examples page you stubbed for this docs site.

Cheers.

**Comment by samselikoff** [comment]
Man that would be awesome!

I think some content is better than none, even if we kinda change the organization of it over time. Right now we have that "Main" section then React and Vue, but I'm thinking we should just go ahead and have a "React" section first, and we can sprinkle it with little guides, Basic example, Jest, Cypress, etc.

I believe @ryanto is working on a video for react-testing-library, which could be another guide under React.

Ryan, what do you say Dan create a new "Basic" guide under a React section, we can work with him and get something going while it's fresh in his mind?

Thanks again for the offer to help, excited to start fleshing these out soon!

**Comment by ryanto** [comment]
Hey Dan! I was offline this weekend, catching up on all the issues today.

This sounds fantastic, absolutely! Like Sam said, I'm working on a video for React based off the demo repo.

If you want to sync up feel free to join discord. We can also discuss in your PR (thanks for opening that!)

**Comment by samselikoff** [comment]
Closing now that we have some React guides in place. We set up a new Quickstarts section and have some good guides that others can follow:

- https://miragejs.com/quickstarts/react/development
- https://miragejs.com/quickstarts/react/react-testing-library

The style we landed on is one-page, all the instructions, the kind of guide that someone looking for a quick start could follow from start to finish without referencing any other part of the docs.

Thanks @nadnoslen again for kicking this off!
